cloud_governance/common/clouds/aws/efs/efs_operations.py

```diff
@@ -0,0 +1,80 @@
+from cloud_governance.common.clouds.aws.utils.common_methods import get_boto3_client
+
+
+class EFSOperations:
+
```

Comment:
Can u pls add the class documentation
What the meaning of EFS = Amazon Elastic File System (EFS) and the purpose of this class?

Reply:
ok, this class helps to delete the EFS resources in zombie_cluster.

Reply:
done.